apiary.apib

```diff
@@ -3504,6 +3505,10 @@ Get details for a single subscription
                         + taxable (Money)
                         + tax (Money)
             + `web_url`: `https://focus.teamleader.eu/subscription_detail.php?id=e2314517-3cab-4aa9-8471-450e73449041` (string)
+            + `book_invoices`: true (boolean)
```

Comment:
In hindsight, I think I would've loved to see this as `invoice_status: draft|booked` instead, booleans are evil :)

Reply:
That's what I am talking about @Jdww, we are talking about the same thing.

We want to decide what the initial state of an invoice must be, but what is the state of that invoice if `book_invoices` is set to `false`? We know that it is booked when set to true, but what is the false state? You could say "draft", but how do you know, it's not reflected as such.

Reply:
> That's what I am talking about @Jdww, we are talking about the same thing.
> 
> We want to decide what the initial state of an invoice must be, but what is the state of that invoice if `book_invoices` is set to `false`? We know that it is booked when set to true, but what is the false state? You could say "draft", but how do you know, it's not reflected as such.

We could also say that booking is an action, and it doesn't really matter which other states there are (and we don't support selecting any state).
So, this is **not** about selecting an initial invoice status, it's about whether the invoices should be booked after creation (or not).

Reply:
Conclusion: https://github.com/teamleadercrm/api/pull/649#discussion_r958645094